src/ckanext-ann-arbor/ckanext/ann_arbor/views/dataset.py

```
@@ -0,0 +1,126 @@
+import logging
+
+import ckan.lib.base as base
+import ckan.lib.navl.dictization_functions as dict_fns
+import ckan.logic as logic
+from ckan.common import _, config, request
```

Ruff (0.11.9)

`ckan.common.config` imported but unused

Remove unused import: `ckan.common.config`

(F401)